### ISSUE TITLE
Improve screen reader support

### ISSUE THREAD

**Issue by usame-algan** [safe-global/safe-wallet-web#2359]
## What is the feature about

The UI doesn't have proper screen reader support.

## The list of requirements

- [ ] Don't read out loud every character of an address
